src/core/server/i18n/get_translation_paths.ts

```diff
@@ -18,7 +18,7 @@ const I18N_RC = '.i18nrc.json';
 
 export async function getTranslationPaths({ cwd, nested }: { cwd: string; nested: boolean }) {
   const glob = nested ? `*/${I18N_RC}` : I18N_RC;
-  const entries = await globby(glob, { cwd });
+  const entries = await globby(glob, { cwd, dot: true });
```

Comment:
__note:__ I believe it's not strictly necessary in this case since pattern already includes dot-file name, but it feels safer to be explicit here.